client/src/legacy/LeftAndMain.Tree.js

```diff
@@ -88,7 +88,13 @@ $.entwine('ss.tree', function($){
               }
             }
           })
-          .bind('move_node.jstree', function(e, data) {
+          .bind('move_node.jstree', async function(e, data) {
+            if (!await self.canMove(data)) {
```

Comment:
Where is this promise defined?

Reply:
I promise it is extended in https://github.com/silverstripe/silverstripe-cms/pull/2470/files#diff-126d72786d20ac82ea0f3445505130a7R146